tests/unit_tests/test_ladder_service.py

```diff
@@ -67,18 +67,26 @@ async def test_load_from_database(ladder_service, queue_factory):
             Map(id=3, name="SCMP_003", path="maps/scmp_003.zip"),
         ]
 
+        queue = ladder_service.queues["neroxis1v1"]
+        assert queue.name == "neroxis1v1"
+        assert len(queue.map_pools) == 1
+        assert list(queue.map_pools[4][0].maps.values()) == [
+            NeroxisGeneratedMap.of({"version": "0.0.0", "spawns": 2, "size": 512, "type": "neroxis"}),
+        ]
+
 
 @fast_forward(5)
 async def test_load_from_database_new_data(ladder_service, database):
     async with database.acquire() as conn:
-        result = await conn.execute(matchmaker_queue.insert().values(
+        await conn.execute(matchmaker_queue.insert().values(
+            id=5,
```

Comment:
Oof that's annoying. Maybe make the id sufficiently large here that we won't need to worry about updating this test every time a new queue is added.